jcl/src/java.base/share/classes/java/lang/Class.java

```diff
@@ -2131,26 +2134,35 @@ private static String getNonArrayClassPackageName(Class<?> clz) {
  * @see			#getPackage
  */
 /*[IF Sidecar19-SE]*/
-public 
+public
 /*[ENDIF] Sidecar19-SE */
 String getPackageName() {
+	String packageName = this.packageNameString;
+	if (null == packageName) {
 /*[IF Sidecar19-SE]*/
-	if (isPrimitive()) {
-		return "java.lang"; //$NON-NLS-1$
-	}
-	if (isArray()) {
-		Class<?> componentType = getComponentType();
-		while (componentType.isArray()) {
-			componentType = componentType.getComponentType();
+		if (isPrimitive()) {
+			packageName = "java.lang"; //$NON-NLS-1$
 		}
-		if (componentType.isPrimitive()) {
-			return "java.lang"; //$NON-NLS-1$
-		} else {
-			return getNonArrayClassPackageName(componentType);
+		else if (isArray()) {
```

Comment:
nitpick: `} else if (isArray()) {`

Reply:
Changed in https://github.com/eclipse-openj9/openj9/compare/c1fc6e25e9028ddb5a58776f35ee9d598f2d46a2..55dd62c6660f6b9165f0b3edf80e54ea799de24c